onnxruntime/core/providers/op_kernel_type_control_utils.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
copied from https://github.com/microsoft/onnxruntime/pull/6547
will merge after that PR is merged